logos/farmsby/farmsby.svg

```diff
@@ -0,0 +1,19 @@
+<svg width="153" height="43" viewBox="0 0 153 43" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0)">
```

Comment:
Nice @prezine
Looks like there's some more room for optimisation here.

Ensure your SVG & PNG logos have been properly optimised with tools like [SVGOMG](https://jakearchibald.github.io/svgomg/) & [Squoosh](https://squoosh.app/) (OxiPNG works well) respectively